packages/core/src/ts-patch/install-patch.ts

```diff
@@ -0,0 +1,19 @@
+import fs from "node:fs";
+import { createRequire } from "node:module";
+
+import { transformTscContent } from "./shared.js";
+
+const require = createRequire(import.meta.url);
+const typescriptPath = require.resolve("typescript");
+
+const origReadFileSync = fs.readFileSync;
```

Comment:
[Naming] I kept reading this as "origami" for some reason. Abbreviations are hard. 😄 

```suggestion
const originalReadFileSync = fs.readFileSync;
```